doc/communication_team.rst

```diff
@@ -11,6 +11,10 @@
     </div>
     <div>
     <a href='https://github.com/francoisgoupil'><img src='https://avatars.githubusercontent.com/u/98105626?v=4' class='avatar' /></a> <br />
-    <p>Francois Goupil</p>
+    <p>francoisgoupil</p>
     </div>
+    <div>
+    <a href='https://github.com/reshamas'><img src='https://avatars.githubusercontent.com/u/2507232?v=4' class='avatar' /></a> <br />
+    <p>Reshama Shaikh</p>
```

Comment:
Reshama is still in the comm team on github. Should we remove her from the comm team ?
ref https://github.com/scikit-learn/scikit-learn/pull/22942

Reply:
I think that it is fine to keep her. She may still pop in and help with the blog for instance.

Reply:
Ok, I've hard coded that. She's still in the comm team but won't show up on the page